packages/ts/react-auth/src/ProtectedRoute.tsx

```diff
@@ -42,6 +42,7 @@ export function ProtectedRoute({ redirectPath, access, element }: ProtectedRoute
 
   return element;
 }
+ProtectedRoute.type = 'ProtectedRoute'; // This is for copilot to recognize this
```

Comment:
Should this also still be exported?

Reply:
Exporting was added in https://github.com/vaadin/hilla/pull/2347 to solve the same issue. It no longer needs to be exported for copilot to work